### PR TITLE
Fix the cleanup of cached gems

### PR DESCRIPTION
**Pull request by tas50**
Make sure we expand the path first so this works

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: omnibus/config/software/more-ruby-cleanup-supermarket.rb [hunk 1]
```diff
@@ -26,11 +26,13 @@
...
+    remove_directory "#{install_dir}/embedded/service/supermarket/vendor/cache"
 
-    remove_directory "#{vendor_dir}/bundle/ruby/*/build_info"
-    remove_directory "#{vendor_dir}/bundle/ruby/*/cache"
-    remove_directory "#{vendor_dir}/bundle/ruby/*/doc"
-    remove_directory "#{vendor_dir}/cache"
+    # this expands into the appropriate Ruby release number dir
+    vendor_ruby_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/bundle/ruby/*/")
+
+    remove_directory "#{vendor_ruby_dir}//build_info"
```
Comment (RajeshPaul38) [inline omnibus/config/software/more-ruby-cleanup-supermarket.rb:34]: Should be single slash(/).
```suggestion
    remove_directory "#{vendor_ruby_dir}/build_info"
```

File: omnibus/config/software/more-ruby-cleanup-supermarket.rb [hunk 2]
```diff
@@ -26,11 +26,13 @@
...
   block "Delete bundler git cache, docs, and build info" do
-    vendor_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/")
+    remove_directory "#{install_dir}/embedded/service/supermarket/vendor/cache"
 
-    remove_directory "#{vendor_dir}/bundle/ruby/*/build_info"
-    remove_directory "#{vendor_dir}/bundle/ruby/*/cache"
-    remove_directory "#{vendor_dir}/bundle/ruby/*/doc"
-    remove_directory "#{vendor_dir}/cache"
+    # this expands into the appropriate Ruby release number dir
+    vendor_ruby_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/bundle/ruby/*/")
```
Comment (RajeshPaul38) [inline omnibus/config/software/more-ruby-cleanup-supermarket.rb:32]: remove trailing slash(/) as the paths below includes slash(/)
```suggestion
    vendor_ruby_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/bundle/ruby/*")
```

**Review by RajeshPaul38 — CHANGES_REQUESTED** [review]
Have added a few suggestions.

**Comment by tas50** [comment]
Thanks @RajeshPaul38

**Pull request merged**